tests/gis_tests/geoapp/test_functions.py

```diff
@@ -223,12 +223,9 @@ def test_intersection(self):
         geom = Point(5, 23, srid=4326)
         qs = Country.objects.annotate(inter=functions.Intersection('mpoly', geom))
         for c in qs:
-            if spatialite or mysql:
-                # When the intersection is empty, Spatialite and MySQL return None
+            if spatialite or mysql or oracle:
```

Comment:
Yes, I thought about this, but I'm planning to do some adjacent changes and I think that it would more clear after that what feature flag we need to add.